.lintrc

```diff
@@ -1,6 +1,7 @@
 # See https://github.com/markstory/lint-review
 [files]
 ignore = */migrations/*
```

Comment:
Yeah we still need it for migrations whenever we update to 1.7. At that point we could delete all the files in south_migrations and remove it from this file though.